esmvaltool/recipes/examples/recipe_check_obs.yml

```diff
@@ -753,6 +753,21 @@ diagnostics:
     scripts: null
 
 
+  CLARA-AVHRR:
+    description: CLARA-AVHRR check
+    variables:
+      clt:
+        mip: Amon
+      clivi:
+        mip: Amon
+      clwvi:
```

Comment:
lwp?

Reply:
Yes, you are right. Same as above. Thanks for spotting!